FactGenerator/src/InstructionVisitor.cpp

```diff
@@ -687,6 +687,26 @@ void InstructionVisitor::visitShuffleVectorInst(
   writeInstrOperand(pred::shufflevector::mask, iref, SVI.getOperand(2));
 }
 
+void InstructionVisitor::visitUnaryInstruction(
+    const llvm::UnaryInstruction &UI) {
+  switch (UI.getOpcode()) {
+    default:
```

Comment:
tiny nit: I find it kind of counter-intuitive to have the default case first, probably because it's the opposite of how you'd do it in Haskell/Rust with pattern-matching. This doesn't have to change, just a note.